tests/gaia/ShowVersion/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,10 @@
+expected_output = {
+    'version': {
```

Comment:
from key/value pair perspective, it'll be better to have version number under `version` like:
```
    'version': {
        'R80.40': {
```
There might be output variations, but based on given output, I thought we can simplify like below:
```
{
    'version': 'R80.40',
    'os_build': '294',
    'os_kernel': '3.10.0-957.21.3cpx86_64',
    'os_edition': '64-bit',
}

Reply:
Agreed, that proposed schema is much more clear. Schema and associated tests have been updated.